### PR TITLE
feat(screenshare): picture-in-picture

### PR DESCRIPTION
**Pull request by JoVictorNunes**
### What does this PR do?

This PR implements Picture-in-Picture mode for the screenshare.

Most of the desktop browsers supports picture-in-picture, all but Internet Explorer and Firefox. On mobile, only Safari supports it.

**Supported browsers:**

Desktop:
- Chrome (>=69)
- Edge (>=79)
- Opera (>=56)
- Safari (>=13.1)

Mobile:
- Safari (>=13.4)

### Motivation

Picture-inPicture can improve the productivity of the users. You can see the screen sharing of the presenter while performing any other tasks.

### Presenter:

Captura de tela de 2021-12-02 11-36-17

### Other Participants:

Captura de tela de 2021-12-02 11-31-56

Captura de tela de 2021-12-02 11-32-05

Captura de tela de 2021-12-02 11-32-15

Captura de tela de 2021-12-02 11-32-24

### CODE REVIEW

**Comment by antobinary** [comment]
That's pretty neat!

**Comment by eegcl-bugs** [comment]
Hi, I think also the other way round such a feature would be very very important.

So that if someone holds a lecture, the shared presentation is big and the video of the presenting person is small in the bottom right corner on top of the presentation.

Like shown here:
[https://cdn.posteo.de/news/2021/01041700/ccc-screenshot.jpeg](url)

File: bigbluebutton-html5/imports/ui/components/pip-button/component.jsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+          isInPictureInPictureMode
+            ? (
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 24 24"
+                width="24"
+                height="24"
+              >
+                <path fill="none" d="M0 0h24v24H0z" />
+                <path d="M21 3a1 1 0 0 1 1 1v7h-2V5H4v14h6v2H3a1 1 0 0 1-1-1V4a1 1 0 0 1 1-1h18zm0 10a1 1 0 0 1 1 1v6a1 1 0 0 1-1 1h-8a1 1 0 0 1-1-1v-6a1 1 0 0 1 1-1h8zm-9.5-6L9.457 9.043l2.25 2.25-1.414 1.414-2.25-2.25L6 12.5V7h5.5z" fill="rgba(255,255,255,1)" />
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/pip-button/component.jsx:76]: Seems these two icons should be added in https://github.com/bigbluebutton/bigbluebutton-font and included in the general .svg .woff ...
Do you remember where you got them from - are they "libre"?

Comment (JoVictorNunes) [reply bigbluebutton-html5/imports/ui/components/pip-button/component.jsx:76]: Yes, they are from https://github.com/Remix-Design/RemixIcon that is under Apache-2.0 License.

Comment (JoVictorNunes) [reply bigbluebutton-html5/imports/ui/components/pip-button/component.jsx:76]: @antobinary How can I proceed?

Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/pip-button/component.jsx:76]: I am clarifying this process with the bigbluebutton-fonts experts. I'll update when it's clear

**Comment by JoVictorNunes** [comment]
> Is this pull request closing #13180 ?

Not fully. This PR implements Pic-in-Pic for screensharing, but not for webcams. So I'd say it partially closes https://github.com/bigbluebutton/bigbluebutton/issues/13180.

**Pull request closed without merging**